### ISSUE TITLE
Flock wants to import circle contacts

### ISSUE THREAD

**Issue by markfaine** [WhisperSystems/Flock#96]
I'm trying ot import from Google the contacts "My Contacts" but flock is importing over 250+ people from Google Circles.  I've tried every way I can think of to prevent these circle members from being included as contacts.  I only want the contacts in "My Contacts" but can't figure out how to make this happen.  I'm not even sure where Flock is getting its count since it doesn't correspond to the number of users in circles, the number of users in the My Contacts, or even the sum of the two.  Any help would be appreciated.

**Comment by rhodey** [comment]
hey @markfaine, I'll try and setup google circles on an account of mine later this week and see what I can to reproduce. it is very likely that in order to prevent this from happening a setting will have to be added to Flock because I expect that some people might prefer this behavior. If you could suggest a possible UX for such a setting that would be helpful. I'd probably put it somewhere in the import account UI conditional on a google account being synced.

**Comment by markfaine** [comment]
Yeah, I guess something like Groups: and a tree hierarchy like structure
that has "My Contacts" at the top and each of the groups found on the
account as a selection option within it.  A separate control for "Include
Circles".

For example:

*Groups:*
▢ My Contacts
    ▢  Friends
    ▢  Family
    ▢  Coworkers

*Include Circles:*  <slider>

Thanks
